.storybook/preview.js

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
@arya56 can you remove this file? Unless there is a reason for it to be here. :)

Reply:
Done